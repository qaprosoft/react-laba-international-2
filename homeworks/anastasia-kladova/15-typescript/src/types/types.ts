export type BaseOperation = (num1: number, num2: number) => IResult;
export type SqrtOperation = (num: number) => IResult;

export interface IResult {
  value: number | null;
  errorMessage: string;
}

export interface ICalculator {
  add: BaseOperation;
  subtract: BaseOperation;
  multiply: BaseOperation;
  divide: BaseOperation;
  power: BaseOperation;
  sqrt: SqrtOperation;
}
