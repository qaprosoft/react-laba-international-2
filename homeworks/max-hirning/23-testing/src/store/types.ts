export interface ITodo {
  id: string;
  value: string;
  isDone: boolean;
}

interface IActionAddTodo {
  payload: ITodo;
  type: "ADD_TODO";
}

interface IActionSetTodos {
  type: "SET_TODO";
  payload: ITodo[];
}

interface IActionUpdateTodo {
  type: "UPDATE_TODO";
  payload: Pick<ITodo, "id"|"value">;
}

interface IActionToggleTodo {
  type: "TOGGLE_TODO";
  payload: Pick<ITodo, "id">;
}

interface IActionDeleteTodo {
  type: "DELETE_TODO";
  payload: Pick<ITodo, "id">;
}

export interface ITodosStore {
  todos: ITodo[];
  addTodo: (value: string) => void;
  toggleTodo: (id: string) => void;
  deleteTodo: (id: string) => void;
  updateTodo: (id: string, value: string) => void;
}

export type IAction = IActionSetTodos|IActionAddTodo|IActionUpdateTodo|IActionToggleTodo|IActionDeleteTodo;