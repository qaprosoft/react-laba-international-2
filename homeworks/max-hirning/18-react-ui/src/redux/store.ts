import { tilesReducer } from './tilse'
import { configureStore } from '@reduxjs/toolkit'

export const store = configureStore({
  reducer: {
    tiles: tilesReducer,
  },
})

export type AppDispatch = typeof store.dispatch
export type RootState = ReturnType<typeof store.getState>