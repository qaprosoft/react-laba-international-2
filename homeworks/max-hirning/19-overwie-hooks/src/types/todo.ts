export interface ITodo {
  id: string;
  value: string;
  isDone: boolean;
}