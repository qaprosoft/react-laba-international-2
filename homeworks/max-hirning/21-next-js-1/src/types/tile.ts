export interface ITile {
  id: number;
  url: string;
  last_name: string;
  first_name: string;
}