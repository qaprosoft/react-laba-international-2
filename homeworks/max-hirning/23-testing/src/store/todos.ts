import * as React from "react";
import { IAction, ITodo, ITodosStore } from "./types";

export const todosReducer = (state: ITodo[], { type, payload }: IAction) => {
  switch (type) {
    case 'SET_TODO':
      return payload;
    case 'ADD_TODO':
      return [...state, payload];
    case 'UPDATE_TODO':
      return state.map(todo => todo.id === payload.id ? { ...todo, value: payload.value } : todo);
    case 'TOGGLE_TODO':
      return state.map(todo => todo.id === payload.id ? { ...todo, isDone: !todo.isDone } : todo);
    case 'DELETE_TODO':
      return state.filter(todo => todo.id !== payload.id);
    default:
      return state;
  }
};

export const TodosContext = React.createContext<ITodosStore|null>(null);