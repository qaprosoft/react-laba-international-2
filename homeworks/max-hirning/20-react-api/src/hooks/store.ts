import * as React from "react";
import { ITodo } from "../store/types";
import { todosReducer } from "../store/todos";

export function useTodosStore() {
  const [todos, dispatch] = React.useReducer(todosReducer, []);

  const deleteTodo = (id: string): void => {
    dispatch({ type: 'DELETE_TODO', payload: { id } });
  };

  const addTodo = (value: string): void => {
    dispatch({ type: 'ADD_TODO', payload: { id: (Date.now()).toString(), value, isDone: false } });
  };

  const toggleTodo = (id: string): void => {
    dispatch({ type: 'TOGGLE_TODO', payload: { id } });
  };

  const setTodos = (todos: ITodo[]): void => {
    dispatch({ type: 'SET_TODO', payload: todos });
  };

  const updateTodo = (id: string, value: string): void => {
    dispatch({ type: 'UPDATE_TODO', payload: { id, value } });
  };

  return { todos, deleteTodo, addTodo, toggleTodo, updateTodo, setTodos };
}