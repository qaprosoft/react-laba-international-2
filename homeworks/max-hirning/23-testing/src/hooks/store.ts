import * as React from "react";
import { ITodo } from "../store/types";
import { todosReducer } from "../store/todos";

export function useTodosStore() {
  const [state, dispatch] = React.useReducer(todosReducer, []);

  const deleteTodo = (id: string): void => {
    dispatch({ type: 'DELETE_TODO', payload: { id } });
  };

  const addTodo = (value: string): void => {
    dispatch({ type: 'ADD_TODO', payload: { id: (Date.now()).toString(), value, isDone: false } });
  };

  const toggleTodo = (id: string): void => {
    dispatch({ type: 'TOGGLE_TODO', payload: { id } });
  };

  const setTodos = React.useCallback((todos: ITodo[]): void => {
    dispatch({ type: 'SET_TODO', payload: todos });
  }, [])

  const updateTodo = (id: string, value: string): void => {
    dispatch({ type: 'UPDATE_TODO', payload: { id, value } });
  };

  return { todos: state, deleteTodo, addTodo, toggleTodo, updateTodo, setTodos };
}