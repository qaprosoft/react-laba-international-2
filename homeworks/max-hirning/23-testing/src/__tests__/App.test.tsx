import { todosReducer } from "../store/todos";
import TestRenderer from 'react-test-renderer';
import { expect, test, describe } from 'vitest';
import CreateTodoComponent from '../components/CreateTodo';
import TodoElComponent from "../components/TodoEl";
import { ITodo } from "../store/types";

describe.concurrent('Test todos reducer', () => {
  test('Set todos', () => {
    const todos = [
      {
        id: "1",
        value: "Some value",
        isDone: false,
      },
      {
        id: "2",
        value: "Some value 2",
        isDone: false,
      },
      {
        id: "3",
        value: "Some value 3",
        isDone: false,
      }
    ]
    
    expect(todosReducer(todos, { type: "SET_TODO", payload: todos })).toStrictEqual(todos)
  })
  test('Add todo', () => {
    const todos = [
      {
        id: "1",
        value: "Some value",
        isDone: false,
      },
      {
        id: "2",
        value: "Some value 2",
        isDone: false,
      },
      {
        id: "3",
        value: "Some value 3",
        isDone: false,
      }
    ]
    const newTodo = {
      id: "4",
      value: "Some value 4",
      isDone: false,
    }
    
    expect(todosReducer(todos, { type: "ADD_TODO", payload: newTodo })).toStrictEqual([...todos, newTodo])
  })
  test('Update todo', () => {
    const todos = [
      {
        id: "1",
        value: "Some value",
        isDone: false,
      },
      {
        id: "2",
        value: "Some value 2",
        isDone: false,
      },
      {
        id: "3",
        value: "Some value 3",
        isDone: false,
      }
    ]

    expect(todosReducer(todos, { type: "UPDATE_TODO", payload: {id: "3", value: "Hello world"} })).toStrictEqual([
      {
        id: "1",
        value: "Some value",
        isDone: false,
      },
      {
        id: "2",
        value: "Some value 2",
        isDone: false,
      },
      {
        id: "3",
        value: "Hello world",
        isDone: false,
      }
    ])
  })
  test('Toggle todo', () => {
    const todos = [
      {
        id: "1",
        value: "Some value",
        isDone: false,
      },
      {
        id: "2",
        value: "Some value 2",
        isDone: true,
      },
      {
        id: "3",
        value: "Some value 3",
        isDone: false,
      }
    ]
    
    expect(todosReducer(todos, { type: "TOGGLE_TODO", payload: { id: "2" } })).toStrictEqual([
      {
        id: "1",
        value: "Some value",
        isDone: false,
      },
      {
        id: "2",
        value: "Some value 2",
        isDone: false,
      },
      {
        id: "3",
        value: "Some value 3",
        isDone: false,
      }
    ])
  })
  test('Remove todo', () => {
    const todos = [
      {
        id: "1",
        value: "Some value",
        isDone: false,
      },
      {
        id: "2",
        value: "Some value 2",
        isDone: false,
      },
      {
        id: "3",
        value: "Some value 3",
        isDone: false,
      }
    ]
    
    expect(todosReducer(todos, { type: "DELETE_TODO", payload: {id: "1"} })).toStrictEqual([
      {
        id: "2",
        value: "Some value 2",
        isDone: false,
      },
      {
        id: "3",
        value: "Some value 3",
        isDone: false,
      }
    ])
  })
})

test('Input active after button was pressed', () => {
  const component = TestRenderer.create(<CreateTodoComponent />);
  const root = component.root;

  expect(component.toJSON()).toMatchSnapshot(); 

  const input = root.findByType('input');
  input.props.onChange({ target: { value: 'Test Todo' } });

  const button = root.findByType('button');
  button.props.onClick();

  expect(component.toJSON()).toMatchSnapshot();
})
describe.concurrent('Todos el tests', () => {
  const todo: ITodo = {
    id: "1",
    value: "Some value",
    isDone: false,
  }
  
  test('Todos el shouldnt changes with its status changing', () => {
    const component = TestRenderer.create(<TodoElComponent id={todo.id} isDone={todo.isDone} value={todo.value} />);
    const root = component.root;
  
    expect(component.toJSON()).toMatchSnapshot(); 
  
    const input = root.findByType('input');
    input.props.onClick();
  
    expect(component.toJSON()).toMatchSnapshot();
  })

  test('Todos el shouldnt change with delet btn was pressed', () => {
    const component = TestRenderer.create(<TodoElComponent id={todo.id} isDone={todo.isDone} value={todo.value} />);
    expect(component.toJSON()).toMatchSnapshot(); 
    const { rendered } = component.toTree()?.rendered;
    const deleteButton = rendered[1].rendered.find(element => element.props.id === 'delete-btn');
    expect(deleteButton).toBeDefined();
    deleteButton.props.onClick();
    expect(component.toJSON()).toMatchSnapshot(); 
  })
})