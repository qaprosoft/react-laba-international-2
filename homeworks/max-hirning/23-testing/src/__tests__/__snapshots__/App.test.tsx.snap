// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Input active after button was pressed 1`] = `
<div
  className="_container_0f8d1c"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    placeholder="Create Todo-Task"
    style={{}}
    value=""
  />
  <button
    className="_button_0f8d1c"
    disabled={true}
    onClick={[Function]}
    style={
      {
        "cursor": "not-allowed",
        "opacity": 0.5,
      }
    }
  >
    Add
  </button>
</div>
`;

exports[`Input active after button was pressed 2`] = `
<div
  className="_container_0f8d1c"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    placeholder="Create Todo-Task"
    style={{}}
    value=""
  />
  <button
    className="_button_0f8d1c"
    disabled={true}
    onClick={[Function]}
    style={
      {
        "cursor": "not-allowed",
        "opacity": 0.5,
      }
    }
  >
    Add
  </button>
</div>
`;

exports[`Todos el tests > Todos el changes with its status changing 1`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el changes with its status changing 2`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el shouldnt change with delet btn was pressed 1`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el shouldnt change with delet btn was pressed 2`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el shouldnt change with delet btn was pressed 3`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value="Some value"
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el shouldnt change with delet btn was pressed 4`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value="Some value"
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style changes with edit btn was pressed 1`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style changes with edit btn was pressed 2`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style shouldnt change with delet btn was pressed 1`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style shouldnt change with delet btn was pressed 2`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style shouldnt change with edit btn was pressed 1`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style shouldnt change with edit btn was pressed 2`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style shouldnt changes with edit btn was pressed 1`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;

exports[`Todos el tests > Todos el style shouldnt changes with edit btn was pressed 2`] = `
<div
  className="_container_fee394"
>
  <input
    className="_input_141368"
    onChange={[Function]}
    onClick={[Function]}
    placeholder=""
    readOnly={true}
    style={
      {
        "backgroundColor": "#ECF3FF",
        "cursor": "pointer",
        "textDecoration": "auto",
      }
    }
    value=""
  />
  <div
    className="_buttonsGroup_fee394"
  >
    <button
      className="_button_fee394"
      onClick={[Function]}
    >
      <img
        alt="edit todo"
        className="_icon_fee394"
        src="/src/assets/edit.svg"
      />
    </button>
    <button
      className="_button_fee394"
      id="delete-btn"
      onClick={[Function]}
    >
      <img
        alt="delete todo"
        className="_icon_fee394"
        src="/src/assets/delete.svg"
      />
    </button>
  </div>
</div>
`;
